try {
  throw 'my error string'
} catch (error) {
  console.log(error)
}
